### ISSUE TITLE
 multi label `class_weight` 

### ISSUE THREAD

**Issue by Drlilou** [tensorflow/tensorflow#54697]
hi 
i was trying to deal with multi-output classification problem on an imbalanced dataset using class_weight parameter of fit method  ; but i faced this issue; i am looking for an alternative or solution 
`class_weight` is only supported for Models with a single output.
thanks

**Comment by mohantym** [comment]
Hi @Drlilou !
Could you please update the [template](https://github.com/tensorflow/tensorflow/issues/new/choose) too as it will help us analyze the issue [tf version, steps followed before you ran into this error or stand alone code/colab gist to reproduce the issue faced].Thanks!

**Comment by Drlilou** [comment]
tensorflow version : 2.8.0
environnement : colab 
model.fit( x_train,[y_train1,y_train2] , batch size=128, class_weight=....)

**Comment by pat749** [comment]
you can see this documentation https://www.tensorflow.org/tutorials/structured_data/imbalanced_data .
